src/cosmosdb-preview/azext_cosmosdb_preview/commands.py

```diff
@@ -81,3 +79,85 @@ def load_command_table(self, _):
+    cosmosdb_restorable_gremlin_graphs_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableGremlinGraphsOperations.{}',
+        client_factory=cf_restorable_gremlin_graphs)
+
+    cosmosdb_restorable_gremlin_resources_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableGremlinResourcesOperations.{}',
+        client_factory=cf_restorable_gremlin_resources)
+
+    # restorable table apis sdk
+    cosmosdb_restorable_tables_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableTablesOperations.{}',
+        client_factory=cf_restorable_tables)
+
+    cosmosdb_restorable_table_resources_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableTableResourcesOperations.{}',
+        client_factory=cf_restorable_table_resources)
+
+    # define commands
+    # Restorable apis for sql,mongodb,gremlin and table
+    with self.command_group('cosmosdb', cosmosdb_sdk, client_factory=cf_db_accounts) as g:
+        g.custom_command('restore', 'cli_cosmosdb_restore', is_preview=True)
+        g.custom_command('create', 'cli_cosmosdb_create', is_preview=True)
+        g.custom_command('update', 'cli_cosmosdb_update')
+        g.custom_command('list', 'cli_cosmosdb_list')
+        g.show_command('show', 'get')
+
+    with self.command_group('cosmosdb sql restorable-container', cosmosdb_restorable_sql_containers_sdk, client_factory=cf_restorable_sql_containers, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb mongodb restorable-collection', cosmosdb_restorable_mongodb_collections_sdk, client_factory=cf_restorable_mongodb_collections, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb gremlin restorable-database', cosmosdb_restorable_gremlin_databases_sdk, client_factory=cf_restorable_gremlin_databases, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb gremlin restorable-graph', cosmosdb_restorable_gremlin_graphs_sdk, client_factory=cf_restorable_gremlin_graphs, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb gremlin restorable-resource', cosmosdb_restorable_gremlin_resources_sdk, client_factory=cf_restorable_gremlin_resources, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb table restorable-table', cosmosdb_restorable_tables_sdk, client_factory=cf_restorable_tables, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb table restorable-resource', cosmosdb_restorable_table_resources_sdk, client_factory=cf_restorable_table_resources, is_preview=True) as g:
+        g.command('list', 'list')
```

Comment:
We don't create a command group with single command inside. How about `cosmosdb table list-restorable-table/list-restorable-resource`

Reply:
Hi @Yishi, we have similar command group in cli already for sql and mongodb. I kept the similar semantics for tables and gremlin in cli-extension as well.

Reply:
If i change the semantics for them in cli-extension, it'll have mismatch with command semantics on prod cli.